pydevd.py

```diff
@@ -1073,17 +1073,6 @@ def get_file_type(self, frame, abs_real_path_and_basename=None, _cache_file_type
             return _cache_file_type[cache_key]
         except:
             if abs_real_path_and_basename[0] == "<string>":
-                # TODO: This isn't ideal. We should make it so that "<string>" is
```

Comment:
We don't need this anymore after eliminating the internal lambda in `_pydevd_sys_monitoring.py`